src/Meshes/interval.jl

```diff
@@ -123,7 +123,7 @@ function IntervalMesh(
     cmax = Geometry.component(domain.coord_max, 1)
     R = cmax - cmin
     h = stretch.H / R
-    η(ζ) = -h * log1p((expm1(-1 / h)) * ζ)
+    η(ζ) = ζ == 1 ? ζ : -h * log1p((expm1(-1 / h)) * ζ)
```

Comment:
Can these changes, if intentional, be moved to a separate PR?

Reply:
This changes were add by @simonbyrne , to fix a `NaN` issue in the ExponentialStretchedGrid code!